lely_core_libraries/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ ExternalProject_Add(upstr_lely_core_libraries    # Name for custom target
   UPDATE_COMMAND
   COMMAND git reset --hard
   COMMAND git apply --whitespace=fix --reject ${CMAKE_CURRENT_SOURCE_DIR}/patches/0001-Fix-dcf-tools.patch
+  COMMAND git apply --whitespace=fix --reject ${CMAKE_CURRENT_SOURCE_DIR}/patches/0002-Allow-base16-values-for-SupportedObjects.patch
```

Comment:
I would add a comment above with the lely MR link. That way it's more clear when this patch can be cleaned up again.